conans/client/cmd/revisions.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+from conans.errors import NotFoundException
+from conans.util.dates import from_timestamp_to_datetime
+
+
+def get_local_recipe_revision(ref, cache):
```

Comment:
Maybe ``search_revisions``? What is in the ``cmd`` subfolder, is intended to be closely related to a command. There is also a ``conans.search`` package, might be another good location.

Reply:
I'm not searching for anything. Just get the revision of a reference, I will move the functions.